relay-server/src/utils/rate_limits.rs

```diff
@@ -246,9 +255,23 @@ pub struct Enforcement {
     profiles: CategoryLimit,
     /// The combined replay item rate limit.
     replays: CategoryLimit,
+    /// Metrics extraction from a transaction is rate limited.
+    event_metrics: CategoryLimit,
 }
 
 impl Enforcement {
+    /// Returns true if the event should be rate limited.
```

Comment:
```suggestion
    /// Returns `true` if the event should be rate limited.
```

to keep it consistent